spec/std/file_spec.cr

```diff
@@ -234,11 +262,11 @@ describe "File" do
 
   describe "same?" do
     it "compares following symlinks only if requested" do
-      file = File.expand_path(datapath("test_file.txt"))
-      other = File.expand_path(datapath("test_file.ini"))
+      file = File.real_path(datapath("test_file.txt"))
+      other = File.real_path(datapath("test_file.ini"))
 
       with_tempfile("test_file_symlink.txt") do |symlink|
-        File.symlink(file, symlink)
+        File.symlink(to_windows_path(file), symlink)
```

Comment:
Everything handles unix-style paths on windows apart from the *target* of `File.symlink`. Because `Path` PR isn't merged yet, path handling remains broken in windows, so hacks like this are required to get specs to run.